src/hotspot/share/oops/compressedKlass.inline.hpp

```diff
@@ -51,7 +51,7 @@ inline narrowKlass CompressedKlassPointers::encode_not_null(Klass* v, address na
   assert(!is_null(v), "klass value can never be zero");
   assert(check_alignment(v), "Address not aligned");
   uint64_t pd = (uint64_t)(pointer_delta(v, narrow_base, 1));
-  assert(KlassEncodingMetaspaceMax > pd, "change encoding max if new encoding");
+  assert(KlassEncodingMetaspaceMax > pd, "change encoding max if new encoding (Klass " PTR_FORMAT ", Base " PTR_FORMAT ")", p2i(v), p2i(narrow_base));
```

Comment:
Would it be possible to leave this hunk out? I am in the process of rewriting large parts of this for Lilliput, so it won't survive in this current form. These things will be a lot better tested, and the assertion messages make more sense, too)

Reply:
Thanks for taking a look! It would be possible to revert this part. But I don't understand the benefit. Wouldn't it be better to have this extra output for the time being? I don't know when Lilliput will make it to jdk head. It seems to be not targeted, yet.
Or will your new code be ready soon and contributed separately?

Reply:
Okay, sure, maybe you are right. Merging it should be easy enough.